model/eventGroupModel.js

```diff
@@ -3,19 +3,19 @@ const Mongoose = require('mongoose');
 const eventGroupSchema = Mongoose.Schema({
     eventID: {
         type: Mongoose.Types.ObjectId,
-        required: true
+        required: [true, 'Event ID must be specified']
     },
     companyID: {
         type: Mongoose.Types.ObjectId,
-        required: true
+        required: [true, 'Company ID must be specified']
     },
     name: {
         type: String,
-        required: true
+        required: [true, 'Group name must be specified']
     },
     users: {
         type: Array,
```

Comment:
You could create a new Schema for this: `[new Schema({...})]`. Possible attributes would be similar to that of `user` in the `responses` array of `eventQuestion` schema. If you conclude they should be same create a helper schema similator to how `locationSchema` is done.